language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/builder/impl/ImplementFunctionsCodeAction.java

```diff
@@ -128,7 +96,7 @@ private static CodeAction getNoImplementationFoundCommand(Diagnostic diagnostic,
 
         if (!edits.isEmpty()) {
             String commandTitle = CommandConstants.IMPLEMENT_FUNCS_TITLE;
-            return createQuickFixCodeAction(commandTitle, edits, uri);
+            return Collections.singletonList(createQuickFixCodeAction(commandTitle, edits, uri));
```

Comment:
We might need to rethink this bit wider, along with the latest class and object changes

Reply:
Yes, these are already disabled by the code-action provider.